pyproject.toml

```diff
@@ -35,18 +35,19 @@ documentation = "https://softwaremill.github.io/FoXAI/"
 "Bug Tracker" = "https://github.com/softwaremill/foxai/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.7.2,<3.11"
+python = ">=3.8.0,<3.11"
 numpy = ">=1.21.4,<2.0.0"
 captum = ">=0.5.0,<1.0.0"
-torch = ">=1.12.1,<2.0.0"
-pytorch-lightning = ">=1.5.0,<2.0.0"
+torch = ">=2.0.0,<=2.0.1"
+pytorch-lightning = ">=2.0.0,<=2.0.4"
 wandb = ">=0.13.7,<1.0.0"
-pandas = ">=1.1.0,<2.0.0"
+pandas = ">=2.0.0,<=2.0.3"
 single-source = ">=0.3.0,<1.0.0"
 opencv-python = ">=4.7.0.68,<5.0.0"
 hydra-core = ">=1.3.1,<2.0.0"
 seaborn = "^0.12.2"
 requests = "^2.31.0"
+jupyter = "^1.0.0"
```

Comment:
I think ``jupyter`` is not an essential part of our library (it has more of a supplementary role to running examples or using the library), but this should not be a dependency of our library.

Reply:
Agree.